### ISSUE TITLE
No error and cant get Nlog to log any message

### ISSUE THREAD

**Issue by xsoul** [harouny/NLog.Extensions.AzureTableStorage#29]
Hi, 
Here is my config at app.config:

```

<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <extensions>
      <add assembly="NLog.Extensions.AzureTableStorage" />
    </extensions>
    <targets>
      <target name="BlobStorage" xsi:type="File" lineEnding="Default" autoFlush="true" keepFileOpen="false" 
              concurrentWrites="true" fileName="${environment:variable=NLOGDIR}\ApplicationLog.log" 
              archiveFileName="${environment:variable=NLOGDIR}\archive\ApplicationLog_${date:format=yyyyMMdd_hhmm}_{#####}.log" 
              archiveEvery="Minute" archiveNumbering="Sequence" maxArchiveFiles="720"
              layout="${longdate}[${threadid}]${level:uppercase=true} ${logger} ${message} ${exception:format=tostring}"></target>
      <target name="AzureTableStorage2" xsi:type="AzureTableStorage"
              connectionStringKey="StorageAccountConnectionString"
              tableName="msberrorlogs"></target>
    </targets>
    <rules>
      <logger name="*" minlevel="Trace" writeTo="BlobStorage" />
      <logger name="*" minlevel="Error" writeTo="AzureTableStorage2"/>
    </rules>
  </nlog>
```

and here is my appSettings:

```
<add key="StorageAccountConnectionString" value="UseDevelopmentStorage=true"/>
```

but i do not get any error message or exception. Once i remove the AzureTableStorage as nlog target, everything work well (i can log with the File type target). 

i am using development emulator and here is my cloud service settings:

```

<ConfigurationSettings>
      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="UseDevelopmentStorage=true" />
      <Setting name="StorageAccountConnectionString" value="UseDevelopmentStorage=true" />
</ConfigurationSettings>

```

thanks

**Comment by geetar85** [comment]
You might be having same issue as [#13](https://github.com/harouny/NLog.Extensions.AzureTableStorage/issues/13) - I updated the Microsoft.WindowsAzure.ConfigurationManager and started working for me.

**Comment by harouny** [comment]
Should have been fixed in v 1.0.10. Please reopen if its reproducible in v 1.0.10.
